packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -6,153 +6,68 @@ import classnames from 'classnames';
 /**
```

Comment:
As shown by the diff in this file, there's probably more cleanup that we can do, like the removal of `postContentAttributes` from the server.

Reply:
I think we still need `postContentAttributes` to be passed in from the server, otherwise we lose the post content layout for users without the capability of editing raw templates.

Reply:
We can probably remove that by making a request using the "view" context for the template.

Reply:
yeah, I'm thinking there's something to be done on the REST API level though. Ideally, I'd like us to avoid adding more and more settings that get passed as inline script from server to client, ideally we should try to decouple the server from the client as much as we can and only use REST API for data.

Reply:
Sounds good to me 👍